### DISCUSSION TITLE
bearer token is encoded to base64

### DISCUSSION THREAD

**Discussion by gabifericean** [Cornutum/tcases#241] [Q&A] [answered]
Hello,
I started to use this tool for last week and I encounter an issue related to the bearer authentication.

I am generating the tests from an open api definition where is specified a security scheme based on bearer token. The tests are successfully generated, but the bearer token is encoded to base 64. Due to this, I cannot authenticate on the server.
Do you know how I can disable the base 64 encoding for the bearer token?

The open api definition is attached. 
[openapi_iam.zip](https://github.com/Cornutum/tcases/files/9470448/openapi_iam.zip)
![image](https://user-images.githubusercontent.com/6602262/187932582-bb266e39-d645-421b-802a-712e94f43d7a.png)

Thanks!

**Comment by kerrykimbrough** [comment, marked as answer]
Reading the OAuth standard ([RFC 6750](https://datatracker.ietf.org/doc/html/rfc6750)), I understood that base64 encoding was required for the Authorization token. It now seems that was wrong, although it appears I was not the only one who was misled. 

Thanks for reporting this problem. Unfortunately, there is no work-around. A source fix will be required. Look for a new release with this fix soon.

**Reply by kerrykimbrough** [reply]
Fixed in version 3.8.3
